salt/modules/zypperpkg.py

```diff
@@ -2018,6 +2018,73 @@ def purge(name=None, pkgs=None, root=None, **kwargs):  # pylint: disable=unused-
     return _uninstall(name=name, pkgs=pkgs, root=root)
 
 
+def list_holds(pattern=None, full=True, root=True, **kwargs):
+    """
+    List information on locked packages.
+
```

Comment:
This function needs a `.. versionadded:: Phosphorus` line in the docs

Reply:
If you want this to go in for Phosphorus, we need to have versionadded. But it's 3005 now so:
```
.. versionadded:: 3005
```